Dependencies/xr/Source/OpenXR/XR.cpp

```diff
@@ -252,6 +254,33 @@ namespace xr
             std::vector<Frame::Plane> Planes{};
             std::vector<FeaturePoint> FeaturePointCloud{};
         } ActionResources{};
+            
+        struct HandJointData {
```

Comment:
nit: inconsistent formatting. Some of the opening braces are on same line, some are on next line.